modular_skyrat/modules/automapper/code/area_spawn_entries.dm

```diff
@@ -42,7 +42,7 @@
 	mode = AREA_SPAWN_MODE_HUG_WALL
 
 /datum/area_spawn/gbp_machine
-	target_areas = list(/area/station/cargo/lobby, /area/station/cargo/boutique, /area/station/construction/storage_wing) // lmao imagine map standardization
+	target_areas = list(/area/station/cargo/lobby, /area/station/cargo/boutique, /area/station/construction/storage_wing, /area/station/hallway/primary/port /* for kilo */) // lmao imagine map standardization
```

Comment:
You don't need this either. You can map it directly somewhere onto the station. This is just a relic of when kilo was maintained by upstream

Reply:
To be honest, for time being I intend to keep it downstreamed from bubber and just copy whatever you guys do 1:1 so that it doesn't need any active effort.

Reply:
Alright